src/pages/Browse/index.spec.tsx

```diff
@@ -83,6 +84,12 @@ describe(__filename, () => {
       _viewVersionFile,
     };
 
+    // Sometimes we need to mock `dispatch` to prevent it from dispatching
+    // actions which would result in an unhandled promise rejection.
```

Comment:
How about this ?
```suggestion
    // Some tests require data to not have been loaded. Mocking dispatch
    // ensures actions that cause us to load data from the API aren't
    // triggered at all (that would raise an unhandled exception inside
    // async code and cause the node process to terminate, as tests mock API
    // responses but the reducers code usually doesn't handle API responses
    // being empty).
```

Reply:
LTGM

Reply:
@bobsilverberg are you ok with that comment ? If so I can copy/paste it where to the other files.

Reply:
Went ahead and did it in 3c962063a612040ee810090032374e4c5d1059ff